### DISCUSSION TITLE
Openshift and mesh compatibility test - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3733] [Announcements]
# Openshift and mesh compatibility test - DaoCloud Enterprise

DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote and boost your digital business.

https://docs.daocloud.io/en/mspider/test/openshift.html

**Comment by hanxiaop** [comment]
https://docs.daocloud.io/en/mspider/test/openshift.html#creating-a-private-mesh

private mesh 翻译错误
